libs/server/common/src/lib/interceptors/remove-sensitive-fields.interceptor.ts

```diff
@@ -1,34 +1,55 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
 import {
   CallHandler,
   ExecutionContext,
   Injectable,
   NestInterceptor,
 } from '@nestjs/common';
 import { map, Observable } from 'rxjs';
+import * as _ from 'lodash';
 
 @Injectable()
 export class RemoveSensitiveFieldsInterceptor implements NestInterceptor {
-  intercept(context: ExecutionContext, next: CallHandler): Observable<any> {
-    return next.handle().pipe(
-      map((value) =>
-        Array.isArray(value)
-          ? value.map((item) => ({
-              ...item,
-              password: undefined,
-              refreshToken: undefined,
-              passwordResetToken: undefined,
-              emailResetToken: undefined,
-              activationSecret: undefined,
-            }))
-          : {
-              ...value,
-              password: undefined,
-              refreshToken: undefined,
-              passwordResetToken: undefined,
-              emailResetToken: undefined,
-              activationSecret: undefined,
-            }
-      )
-    );
+  intercept(context: ExecutionContext, next: CallHandler): Observable<unknown> {
+    const removeSensitiveFields = (
```

Comment:
Funktioniert des jetzt o in mehreren Ebenen?

Reply:
jo i hobs rekursiv gmacht